src/app/[locale]/globals.css

```diff
@@ -111,3 +111,12 @@ p {
   line-height: 25px;
   color: var(--color-primary);
 }
+
+.expert-card {
```

Comment:
why should this be a css class, rather then svg file?

Reply:
There could be hundreds of users and a image would be rendered for each one, so I thought this method would be more efficient in terms of optimization.

Reply:
Frankly, I cant see how. In this approach you just make css file bigger, meaning it will take more time for initial page to load. You can put svg file in public folder, making it static, so it will be loaded on-demand by browser (most likely AFTER the initial css) and cached into the browser, so I don't see benefits here